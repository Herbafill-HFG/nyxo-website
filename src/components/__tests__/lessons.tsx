import React from "react"
import renderer from "react-test-renderer"
import LessonLimited from "../LessonLimited"

const RendersCorrectly = "renders correctly"

describe("Lesson Limited", () => {
  it(RendersCorrectly, () => {
    const tree3 = renderer.create(<LessonLimited />).toJSON()
    expect(tree3).toMatchSnapshot()
  })
})
