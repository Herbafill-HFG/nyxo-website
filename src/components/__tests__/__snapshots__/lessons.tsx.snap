// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lesson Limited renders correctly 1`] = `
Array [
  <div
    className="sc-bdnylx geYECJ"
  />,
  <div
    className="sc-gtssRu jOMRNv"
  >
    <h2>
      To keep reading this Lesson, create a free account.
    </h2>
    <button
      className="sc-dlnjPT jaIWrT"
    >
      <a
        className="sc-hKFyIo kVjuSm"
        href="/me/register"
      >
        Register with Nyxo
      </a>
    </button>
    <p>
      Already have an account? 
      <a
        href="/me/login"
      >
        Login
      </a>
      .
    </p>
  </div>,
]
`;
